web-dev/src/components/DetailScreen/header-section.js

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from "react";
+
+const HeaderSection = ({movie = null}) => {
+
+  // TODO: retrieve bookmark initial state from API
+  const [bookmarked, setBookmark] = useState(false);
```

Comment:
I'll get this to you once I implement it on the home page :)

Reply:
Thanks!